.gitignore

```diff
@@ -118,3 +118,6 @@ dist
 # @axa-fr/react-oidc generated files
 /public/OidcServiceWorker.js
 /public/OidcTrustedDomains.js
+
+#.vscode
```

Comment:
Will leave a full review later but short term: I'd encourage you to add this to your personal global git ignore (the philosophy here is that this ignore line is specific to you personally and shouldn't be reflected in the project at large / it's up to each dev to locally control the files that their IDEs would normally try to insert)

[Here's an example of what my global git ignore looks like](https://github.com/slifty/dotfiles/blob/main/git/gitignore.symlink).  You should be able to put it at ~/.gitignore.

And then you can update your git config to point to that file: https://github.com/slifty/dotfiles/blob/a8f026cd7ccb38b57f1fc35d09a8f0e05c997028/git/gitconfig.symlink#L12